README.md

```diff
@@ -93,6 +93,7 @@ const keys = extractFromFiles([
 
 - `marker`: The name of the internationalized string marker function. Defaults to `i18n`.
 - `keyLoc`: An integer indicating the position of the key in the arguments. Defaults to `0`. Negative numbers, e.g., `-1`, indicate a position relative to the end of the argument list.
+- `useTypescript`: Defaults to `false`.
```

Comment:
Should we respect the case `useTypeScript`?

Reply:
What do you mean ?

Reply:
I believe it's TypeScript over Typescript.

Reply:
Ah yes you right !

Reply:
Done 👍